source/addonStore/dataManager.py

```diff
@@ -220,7 +220,7 @@ def _getCachedAddonData(self, cacheFilePath: str) -> Optional[CachedAddonsModel]
 	_updateFailureDefaultSuggestion = pgettext(
 		"addonStore",
 		# Translators: A suggestion of what to do when fetching add-on data from the store fails and the default metadata URL is being used.
```

Comment:
No suggestion is being made.
```suggestion
		# Translators: Presented when fetching add-on data from the store fails and the default metadata URL is being used.
```